libs/components/code-examples/project.json

```diff
@@ -59,7 +59,13 @@
           "{projectRoot}/src/**/*.ts",
           "{projectRoot}/src/**/*.html"
         ]
-      }
+      },
+      "dependsOn": [
+        {
+          "target": "build",
+          "projects": "manifest"
```

Comment:
The code examples use the `skyux-eslint` ruleset, which uses the manifest, so it needs to be built before code examples can be linted.